### PR TITLE
Ui bugfixes

### PR DESCRIPTION
**Pull request by kfang**

### CODE REVIEW

**Comment by hellochar** [comment]
lg2m

File: the_game_bazaar/views.py [hunk 1]
```diff
@@ -248,17 +249,23 @@ def ajax_change(request):
         user = authenticate(username=request.user.username, password=request.POST['old_pass'])
         if(user is not None and user.is_active):
             #old pass matches, change pass to new one
-            user.set_password(request.POST['new_pass'])
-            user.save()
-            resp['success'] = True
+            if len(request.POST['new_pass']) < 8 or len(request.POST['new_pass']) > 30:
```
Comment (jamesmuerle) [inline the_game_bazaar/views.py]: Change this number to 5 and make it a variable.

File: the_game_bazaar/views.py [hunk 2]
```diff
@@ -248,17 +249,23 @@ def ajax_change(request):
...
+                resp['error'] = 'Passwords must be 8-30 characters'
```
Comment (jamesmuerle) [inline the_game_bazaar/views.py:258]: Obviously change the text too.

File: the_game_bazaar/views.py [hunk 3]
```diff
@@ -295,6 +302,23 @@ def ajax_register(request):
...
+        return HttpResponse(json.dumps(resp), mimetype="application/json")
+
+    # Check username length
+    if len(username) < 4 or len(username) > 20:
+        resp['error'] = 'A username must be 4-20 characters'
+        return HttpResponse(json.dumps(resp), mimetype="application/json")
+
+    # Check password length
+    if len(password) < 8 or len(password) > 30:
+        resp['error'] = 'A password must be 8-30 characters'
```
Comment (jamesmuerle) [inline the_game_bazaar/views.py:322]: Gotta change it here too.

**Pull request merged**